nvdaHelper/remote/ime.cpp

```diff
@@ -108,12 +108,12 @@ DWORD getIMEVersion(HKL kbd_layout, wchar_t* filename) {
 			// Do not know how to extract version number
 			return 0;
 	}
-	DWORD ver_handle;
+	DWORD ver_handle = 0;

```

Comment:
This line can actually be removed, as getFileVersionInfoSize and getFileVersionInfo don't need it. See comments below.